estimation/estimate_wct.py

```diff
@@ -140,20 +120,29 @@ def estimate_LF_chours(
         The number of cores to use, returns the argument n_cores
         if scale_ncores is not set. Otherwise returns the updated ncores.
     """
-    if data.shape[1] != 5:
-        raise Exception("Invalid input data, has to 5 columns. One for each feature.")
-
-    # Use number of steps (i.e. nx * ny * nz) and nt for LF SVR
-    svr_data = np.concatenate(
-        ((data[:, 0] * data[:, 1] * data[:, 2])[:, None], data[:, -2][:, None]), axis=1
-    )
-
-    core_hours = estimate(
-        data,
-        model,
-        model_type,
-        platform_config[const.PLATFORM_CONFIG.LF_DEFAULT_NCORES.name],
-        lf_svr_input_data=svr_data,
+    if data.shape[1] != 6:
+        raise Exception("Invalid input data, has to 6 columns. One for each feature.")
+    #
+    coefficients = {
+        "a": 0.804_038_96,
```

Comment:
I think something is up with your coefficients?

Reply:
Underscores are valid in numbers, the expected use is like commas when writing large numbers. It helps keep track of where in the number you are :P
TL;DR unless the number itself is wrong, this is fine

Reply:
Numbers are awfully precise. But if that's what the Dr ordered...